beyond-io/ClassRater
# Add Review class methods and tests

Modify **models.py** to add ``print_details`` method

Modify **tests.py** to add **Review** class tests:
  - check the data already stored in the database
  - add a valid review
  - create invalid review (validation constraints)
  - call ```__str__``` method
  - call ```print_details``` method

Modify **0006_review_test_data.py**
Because the migration files are executed upon every test
Check if the test image already exist to avoid duplicates

Add **0007_auto_alter_review_field.py** to modify class fields

fix: #60
